src/index.ts

```diff
@@ -37,8 +37,8 @@ logger.info(`Start probe version ${VERSION} in a ${process.env['NODE_ENV'] ?? 'p
 
 function connect() {
 	const worker = {
-		jobs: new Map<string, number>(),
```

Comment:
like before, my stance didn't change on that. A map of Ids is a better way of tracking jobs.